extensions/Oisthebestletter/sb3ai.js

```
@@ -0,0 +1,212 @@
+  const blobURL = URL.createObjectURL(blob);
+
+  try {
+    return await Scratch.external.importModule(blobURL);
+  } finally {
+    URL.revokeObjectURL(blobURL);
+  }
+}
+
+(async () => {
+  function snapshotStage() {
+    const renderer = window.vm.renderer;
```

ESLint

[error] Use Scratch.vm instead of window.vm

(extension/use-scratch-vm)